src/gba/sio/lockstep.c

```diff
@@ -19,7 +35,14 @@ static uint16_t GBASIOLockstepNodeNormalWriteRegister(struct GBASIODriver* drive
 static void _GBASIOLockstepNodeProcessEvents(struct mTiming* timing, void* driver, uint32_t cyclesLate);
 
 void GBASIOLockstepInit(struct GBASIOLockstep* lockstep) {
-	mLockstepInit(&lockstep->d);
+	GBASIOLockstepInit2(lockstep, true);
```

Comment:
Why is this needed?

Reply:
To allow caller to choose to avoid init the base struct again, in case he/she wants to defer the initialization of derived struct such as the code in `MultiplayerController` which calls `mLockstepInit ` in constructor and separately calls `GBASIOLockstepInit ` in `attachGame`